library/VTKExtensions/Rendering/vtkF3DRenderPass.cxx

```diff
@@ -275,7 +275,8 @@ void vtkF3DRenderPass::Blend(const vtkRenderState* s)
 
     ssImpl << "  vec4 mainSample = texture(texMain, texCoord);\n";
 
-#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 2, 20230824)
+// This check is not compatible with VTK 9.3 pre-release version build number < 20230824
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 3, 0)
```

Comment:
I'm sorry but I don't get it. 

https://gitlab.kitware.com/vtk/vtk/-/merge_requests/8577 has not been backported to release and corresponding commits are not present in 9.3.0 afaics.

Reply:
You're right, the CI is failing, let me try something else